packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -819,7 +835,7 @@ class FlutterVmService {
         // with cleaning up.
         return <FlutterView>[];
       }
-      final List<Object> rawViews = response.json['views'] as List<Object>;
+      final List<Object>? rawViews = response.json!['views'] as List<Object>?;
```

Comment:
Why the `!` if the `List` is nullable?

Reply:
because I only now learned that `?[` was a thing...

Reply:
changed to `?[`